src/entries/popup/components/FullScreen/FullScreenContainer.tsx

```diff
@@ -2,20 +2,24 @@ import React from 'react';
 
 import { POPUP_DIMENSIONS } from '~/core/utils/dimensions';
 import { Box } from '~/design-system';
+import { BackgroundColor } from '~/design-system/styles/designTokens';
 
 import { NAVBAR_HEIGHT } from '../Navbar/Navbar';
 
 export function FullScreenContainer({
   children,
+  background,
 }: {
   children: React.ReactNode;
+  background?: string;
```

Comment:
i think you could change `string` for `BackgroundColor` here and you can avoid the `background as BackgroundColor` in line 22

Reply:
and also FullScreenContainer will only accept `BackgroundColor` strings and not random strings so ts can suggest which values to use